src/Skybrud.Umbraco.GridData/Converters/Umbraco/UmbracoGridConverter.cs

```diff
@@ -22,7 +22,7 @@ public class UmbracoGridConverter : IGridConverter {
             value = null;
 
             if (IsEmbedEditor(control.Editor)) {
-                value = GridControlEmbedValue.Parse(control, token);
+                value = GridControlEmbedValue.Parse(control, token as JObject);
```

Comment:
Hey @ArnoldV,

Thanks again for this PR 👍 

From our chat the other day, I understood it as this PR would maintain backwards compatible with 8.1. Should that still be the case?

Casting the token and passing it to the constructor as a `JObject` breaks in 8.1, since the `value` property is a string, and not an object as in 8.2+. So if we should continue to support 8.1, the token should still be passed on to the constructor as `JToken`, and then we can do some type checks inside the constructor.

Reply:
hmm make sense I'll add a different kind of check